source/code/go/src/plugins/oms.go

```diff
@@ -990,11 +992,28 @@ func InitializePlugin(pluginConfPath string, agentVersion string) {
 		} else {
 			Computer = strings.TrimSuffix(ToString(containerHostName), "\n")
 		}
+		// read proxyendpoint if proxy configured
+		ProxyEndpoint = ""
+		proxyConfigPath := pluginConfig["omsproxy_conf_path"]
+		if _, err := os.Stat(proxyConfigPath); err == nil {
+			Log("Reading proxy configuration for Linux from %s", proxyConfigPath)
+			proxyConfig, err := ioutil.ReadFile(proxyConfigPath)
+			if err != nil {
+				message := fmt.Sprintf("Error Reading omsproxy configuration %s\n", err.Error())
+				Log(message)
+				// if we fail to read proxy configuration, then everything will be fall apart including sending  exceptions to AI
+				SendException(message)
+				log.Fatalln(message)
```

Comment:
So if i do a kubectl logs on the pod which is set with invalid proxy, how do i know its crashlooping due to invalid proxy ? Wht would i see in the log that will tell me that proxy is invalid ?